go/api/response_handlers.go

```diff
@@ -31,6 +31,24 @@ func handleStringOrNullResponse(response *C.struct_CommandResponse) string {
 	return handleStringResponse(response)
 }
 
+func handleStringArrayResponse(response *C.struct_CommandResponse) []string {
+	defer C.free_command_response(response)
+	var len []C.long
+	len = append(len, unsafe.Slice(response.array_elements_len, response.array_value_len)...)
+	var slice []string
+	for k, v := range unsafe.Slice(response.array_value, response.array_value_len) {
+		slice = append(slice, convertCharArrayToString(v, len[k]))
+	}
+	return slice
+}
+
+func handleStringArrayOrNullResponse(response *C.struct_CommandResponse) []string {
```

Comment:
Is it in use anywhere?

Reply:
Removing for now as currently it is not in use.